srcpkgs/python3-jupyter_console/template

```diff
@@ -5,7 +5,8 @@ revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-ipython python3-ipython_ipykernel python3-jupyter_client
- python3-prompt_toolkit python3-Pygments python3-traitlets"
+ python3-prompt_toolkit python3-Pygments python3-traitlets
+ python3-platformdirs"
```

Comment:
This package will need a revbump to see the updated dependency.

Reply:
Done.